src/datachain/data_storage/warehouse.py

```diff
@@ -20,8 +20,8 @@
 from datachain.data_storage.schema import convert_rows_custom_column_types
 from datachain.data_storage.serializer import Serializable
 from datachain.dataset import DatasetRecord, StorageURI
+from datachain.lib.func.inner import path as pathfunc
```

Comment:
[Q] What is the difference between `datachain.lib.func` & `datachain.lib.func.inner`? When would you use one and not the other?

Reply:
I have replaced it back from `datachain.lib.func.inner` to `datachain.sql.functions`.

The reason why we use `datachain.sql.functions` is because in some places internally we are using SQLAlchemy directly, and our `Func` wrapper is not compatible with SQLAlchemy :(